engine/common/rpc/convert/convert.go

```diff
@@ -728,6 +728,36 @@ func EventsToMessages(flowEvents []flow.Event) []*entities.Event {
 	return events
 }
 
+func EventsToMessagesFromVersion(flowEvents []flow.Event, version execproto.EventEncodingVersion) ([]*entities.Event, error) {
+	events := make([]*entities.Event, len(flowEvents))
+	for i, e := range flowEvents {
```

Comment:
you won't need this after merging this: https://github.com/onflow/flow-go/pull/4522#pullrequestreview-1506261565, right?